ClosedXML/Excel/Sparkline/XLSparkLineGroup.cs

```diff
@@ -309,9 +306,9 @@ internal IXLSparkline Add(IXLSparkline sparkline)
             SparklineGroups.Remove(sparkline.Location);
 
             if (_sparklines.ContainsKey(sparkline.Location))
-                _sparklines.Remove(sparkline.Location);
-
-            _sparklines.Add(sparkline.Location, sparkline);
+                _sparklines[sparkline.Location] = sparkline;
+            else
+                _sparklines.Add(sparkline.Location, sparkline);
```

Comment:
Only recently I've discovered that `_sparklines[sparkline.Location] = sparkline;` would be enough: it adds a value if key does not exist, otherwise changes the existing one.

Reply:
What?! My whole life has been a lie! Ok, I'm going to add a commit to simplify all those cases.

Reply:
My reaction exactly! 😆